packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Custom.Example.tsx

```diff
@@ -51,27 +52,6 @@ export class OverflowSetCustomExample extends React.PureComponent {
           }
         ]}
         overflowItems={[
-          {
```

Comment:
Is this removal intentional?

Reply:
redundant commands in our example. from an A11y standpoint the menu was confusing. better to just not have that in the example.